### PR TITLE
Filter queries

### PR DESCRIPTION
**Pull request by jmezalon**

### CODE REVIEW

File: backend/db/queries/orderQueries.js [hunk 1]
```diff
@@ -100,3 +100,10 @@ module.exports = {
   getAllOrderForSingleUser,
   allOrdersForGrandma
 };
+
+//const orderSummary = (req, res, next) => {
+
+// const user_id = parseInt(req.params.user_id;
+//
+// db.any(
+//   "SELECT users.id AS user_id, first_name, last_name, email, dishes.user_id AS grandma_id, phone_number, isGrandma, building_number, address, zip_code, dishes.id AS dish_id, name, price, timeframe, orders.id AS order_id, isCompleted  FROM orders JOIN users ON orders.user_id = users.id JOIN dishes ON dishes.id = orders.dish_id WHERE users.id=$1", [user_id]
```
Comment (chrisdl) [inline backend/db/queries/orderQueries.js]: why are we committing commented out code?

File: backend/db/queries/userQueries.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-const db = require("../connection");
-const authHelpers = require("../../auth/helpers");
+const db = require('../connection');
```
Comment (chrisdl) [inline backend/db/queries/userQueries.js]: this is the problem with auto prettier type things. if one of you has it set to `"` and the other one set to `'` then it makes pull requests and commits over inflated.

File: backend/db/queries/orderQueries.js (hunk 1 above)
Comment (chrisdl) [reply backend/db/queries/orderQueries.js]: remember. ideally a PR should only contain code that does one thing. for example Adds/removes a piece of functionality, or fixes a bug, or even adds some comments or cleans up some SCOPED piece of code.

File: backend/db/queries/userQueries.js (hunk 2 above)
Comment (jmezalon) [reply backend/db/queries/userQueries.js]: I see.. I guess we all have to have our settings to be the same to fix this

Comment (chrisdl) [reply backend/db/queries/userQueries.js]: yeah, put your settings in a file that is also committed to the repo, that way it is shared for the repo across your editing environments. for example for eslint you would have a `.eslintrc.json` file.

https://eslint.org/docs/user-guide/configuring

**Pull request merged**